docs/testplan/Distributed-VoQ-Arch-test-plan.md

```diff
@@ -0,0 +1,782 @@
+
+#### Test Case 9. Disruptive Events
+##### Test Objective
+Verify port, router interface, and neighbor recovery after disruptive events.
+##### Test Steps
+* After the following events:
+    * chassis power cycle,
+    * supervisor reboot,
+* Verify, as in the previous test cases:
+    * Local neighbor learning,
+    * remote neighbor learning and route creation
+    * timeout and clear of neighbors
+     
+
+## Router Interface Lifecycle <a name = "ri"></a>
+
+#### Test Case 1.  IP Interface Creation
+##### Test Objective
+Verify Chassis App DB is updated with new interface entry when a new IP Interface is added.
+##### Test Steps
+* Add IP to a previously unconfigured port by adding minigraph configuration to that linecard.
+* Reload the new minigraph and line card.
+* On the line card:
+    * Verify address state in CLI.
+    * Verify interface in ASIC DB
+* On the supervisor card:
+    * Verify the interface is present in the SYSTEM_INTERFACE table of the Chassis App DB.
+    * Verify the OID is unique, and matches the router interface ID in the ASIC DB.
+    * Verify the slot and port are correct.
+* Verify bidirectional traffic to an attached host on the newly created port from local and remote linecards.
+* Repeat with IPv4, IPv6, dual-stack.
+
+#### Test Case 2. Interface Deletion
+##### Test Objective
+Verify Chassis App DB is updated with new interface entry when an IP interface is removed from a port.
+##### Test Steps
+* Remove IP configuration from a previouslyy configured port by removing the minigraph configuration for that port 
+on the linecard minigraph.
+* Reload the new minigraph and line card.
+* On the line card:
+    * Verify address is removed from CLI.
+    * Verify interface is removed from ASIC DB.
+* On the supervisor card:
+    * Verify the interface is removed from the SYSTEM_INTERFACE table of the Chassis App DB.
+* Verify bidirectional traffic to attached host on the port from local and remote ASICs is dropped.
+* Repeat with IPv4, IPv6, dual-stack.
+        
+
+## Host IP Forwarding <a name="ipfwd">
+
+
+### Configuration
+
+Please reference the [T2 topology](https://github.com/Azure/sonic-mgmt/pull/2638/) files topo_t2.yml and testbed-t2.png for network topology and sample IP addresses.  The addresses and VMS below are taken from that example topology.
+
+VMs attached to line card 1 and line card 2 will be used for this test.
+DUT Port A&B are on line card 1, D is on line card 2.
+```
+                      ---------- DUT ----------    
+                      |--- LC1 ---|--- LC2 ---|
+VM01T3   -------------|A          |           |
+                      |         F0|F1        D|------------- VM01T1
+VM02T3   -------------|B     LB1  |   LB2     |
+```
+
+_VM01T3_
+* Loopbacks:
+    * ipv4: 100.1.0.1/32
+    * ipv6: 2064:100::1/128
+* Ethernet:
+    * ipv4: 10.0.0.1/31
+    * ipv6: FC00:2/126
+
+
+_VM02T3_
+* Loopbacks:
+    * ipv4: 100.1.0.2/32
+    * ipv6: 2064:100::2/128

```

Comment:
Suggest to quote the IPv6 address to avoid rendering `:100:` to special emoji in markdown preview. For example:
`2064:100::2/128`.

Reply:
Fixed in commit d6e0